api/managementpb/actions.proto

```diff
@@ -358,103 +359,163 @@ service Actions {
       post: "/v1/management/Actions/Get"
       body: "*"
     };
+    option (grpc.gateway.protoc_gen_swagger.options.openapiv2_operation) = {
+      summary: "Get Action"
+      description: "Gets an result of given Action."
+    };
   }
   // StartMySQLExplainAction starts MySQL EXPLAIN Action with traditional output.
   rpc StartMySQLExplainAction(StartMySQLExplainActionRequest) returns (StartMySQLExplainActionResponse) {
     option (google.api.http) = {
       post: "/v1/management/Actions/StartMySQLExplain"
       body: "*"
     };
+    option (grpc.gateway.protoc_gen_swagger.options.openapiv2_operation) = {
+      summary: "Start MySQL Explain Action"
+      description: "Starts MySQL EXPLAIN Action with traditional output."
+    };
   }
   // StartMySQLExplainJSONAction starts MySQL EXPLAIN Action with JSON output.
   rpc StartMySQLExplainJSONAction(StartMySQLExplainJSONActionRequest) returns (StartMySQLExplainJSONActionResponse) {
     option (google.api.http) = {
       post: "/v1/management/Actions/StartMySQLExplainJSON"
       body: "*"
     };
+    option (grpc.gateway.protoc_gen_swagger.options.openapiv2_operation) = {
+      summary: "Start MySQLExplainJSON Action"
```

Comment:
Maybe for better readability we can use something like `Start 'MySQL Explain JSON' Action`? Otherwise, we will have difficult to understand set of words, although we want our api to be clear for users.

Reply:
Looks good to me